Tests/SwiftJWTTests/TestJWT.swift

```diff
@@ -110,15 +111,43 @@ class TestJWT: XCTestCase {
     }
     
     func check(jwt: JWT, algorithm: String) {
-        XCTAssertEqual(jwt.header.headers.count, 1, "Wrong number of header fields")
-        XCTAssertEqual(jwt.claims.claims.count, 6, "Wrong number of claims")
+        
+        if jwt.header[.kid] != nil {
```

Comment:
This function seems a bit odd... maybe it should be two functions, one that verifies a normal 'test' JWT and one that verifies the MicroProfile type?